src/poliastro/core/propagation/__init__.py

```diff
@@ -733,24 +735,24 @@ def pimienta(k, r0, v0, tof):
     Parameters
     ----------
     k : float
-        Standar Gravitational parameter
-    r0 : array
+        Standar Gravitational parameter.
+    r0 : ~np.array
         Initial position vector wrt attractor center.
-    v0 : array
+    v0 : ~np.array
         Initial velocity vector.
     tof : float
         Time of flight.
 
     Returns
     -------
-    rf: array
-        Final position vector
-    vf: array
-        Final velocity vector
+    rr: ~np.array
+        Final position vector.
+    vv: ~np.array
+        Final velocity vector.
 
     Note
     ----
-    This algorithm was drived from the original paper: http://hdl.handle.net/10477/50522
+    This algorithm was derived from the original paper: https://www.researchgate.net/publication/288133262_Accurate_Kepler_equation_solver_without_transcendental_function_evaluations
```

Comment:
Or better yet, let's put a proper reference:

Pimienta-Penalver, A., and Crassidis, J. "Accurate Kepler Equation Solver without Transcendental Function Evaluations." AAS Jer-Nan Juang Astrodynamics Symposium. College Station, TX, June 2012, AAS Paper #2012-612

Reply:
Not sure what citation style are we using :) Or whether @jorgepiloto used the thesis or the AAS paper:

![Screenshot 2021-07-02 at 07-11-01 Accurate Kepler Equation Solver without Transcendental - Google Scholar](https://user-images.githubusercontent.com/316517/124223814-bcdec400-db04-11eb-9f4d-05ecc3ea079d.png)